app/views/items/new.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= turbo_frame_tag "new_transaction" do %>
```

Comment:
[ ] At the top of the page the user can see the total amount for the category (sum of all of the amounts of the transactions in that category), kindly fix that 🤓.

your transaction page: 👇
 
![transaction](https://github.com/deniodev/Budget-Buddy/assets/62156607/4c6b7439-58cb-402e-8520-d73c6a2b3049)

how it should be like 👇 : 

![design](https://github.com/deniodev/Budget-Buddy/assets/62156607/ef92d978-af14-451e-bdd0-2ada3baacdf8)